### ISSUE TITLE
[Bug]: Switchbot Curtain with FW3.9 seen as Switchbot TH1

### ISSUE THREAD

**Issue by gzone156** [custom-components/ble_monitor#786]
### What happened?

After a FW upgrade of my Switchbot Curtain, it got auto-discovered by the integration as a Switchbot TH1 sensor, causing lots of ERROR logs about temperature spikes (the Curtain doesn't have any temp sensor onboard).

### Sensor type

Switchbot TH1

### Relevant log output

```shell
Questo errore ha avuto origine da un'integrazione personalizzata.

Logger: custom_components.ble_monitor.sensor
Source: custom_components/ble_monitor/sensor.py:707
Integration: Passive BLE monitor (documentation, issues)
First occurred: 19 marzo 2022, 23:38:28 (978 occurrences)
Last logged: 17:25:34

Temperature spike: -103.1 ([*omitted MAC address*])
Temperature spike: -97.2 ([*omitted MAC address*])
Temperature spike: -101.1 ([*omitted MAC address*])
Temperature spike: -85.1 ([*omitted MAC address*])
```

**Comment by Ernst79** [comment]
Will be fixed with the following commit https://github.com/custom-components/ble_monitor/commit/062925c008c2204adc86e5d198072ce62f9e69b3

It is a bit of a guess, this fix. I assume that two bytes in the BLE advertisement of a TH1 are always `0x0054` (as I saw the same numbers here)

https://github.com/bbostock/Switchbot_Py_Meter/blob/c5cb0daff787a0a8c6e519d49797c96a5e6c6816/meters.py#L68

The second assumption is that it is different for the curtain. So, hopefully I don't break down things for TH1 users and hopefully the curtain has different bytes in this position. If not, let me know. 

Fix will be included in the next release.

**Comment by gzone156** [comment]
Just tested and works great for my Curtain. The device is not "recognized" anymore, and the errors are subsequently gone.

Thanks @Ernst79 for the hyperfast fix.

**Comment by Ernst79** [comment]
Fixed in 8.2.0
